dashboard/test/controllers/code_review_comments_controller_test.rb

```diff
@@ -4,78 +4,250 @@ class CodeReviewCommentsControllerTest < ActionController::TestCase
   self.use_transactional_test_case = true
 
   setup_all do
-    @pilot_teacher = create :teacher
-    create(:single_user_experiment, min_user_id: @pilot_teacher.id, name: 'csa-pilot')
+    @project_owner = create :student
+    @project_owner_channel_id = 'encrypted_channel_id'
+    @project_owner_storage_id = 123
+    @project_storage_app_id = 456
+    @project_version_string = 'special_project_version_string'
+
+    @teacher = create :teacher
+    @section = create :section, user: @teacher
+    @another_student = create :student
```

Comment:
I think it might cause issues to assign different permissions to the same account across multiple tests--maybe create `another_student` within individual tests instead?

Reply:
Drone failed b/c I added a last minute validation that I didn't account for in tests :facepalm:, fixed and failing tests passing (locally at least). 

Are there differences between how records are created/destroyed in tests between Drone/test and development that might cause the issues you're describing? I'm doing the same thing with the `@teacher` and `@project_owner` users (creating different follower/section combinations to check different behavior), should I also change how those are set up?

Reply:
ah ok that was mostly a theory, I wasn't sure if the setup_all method would create just one user for all the tests or not. If it passes it's probably fine, but a good thing to double check.